apptools/preferences/preferences_helper.py

```diff
@@ -65,9 +72,23 @@ def _preferences_default(self):
     def _anytrait_changed(self, trait_name, old, new):
         """ Static trait change handler. """
 
+        if self.preferences is None:
+            return
+
+        # If the trait was a list or dict '_items' trait then just treat it as
+        # if the entire list or dict was changed.
+        if trait_name.endswith('_items'):
+            trait_name = trait_name[:-6]
```

Comment:
when we get to python 3.9, we would be able to use `removesuffix` method - which would make the code more readable. ref https://www.python.org/dev/peps/pep-0616/

maybe we should add two backwards compatible `removeprefix` and `removesuffix` functions in a traits utility - which other libraries that depend on traits can use.